### PR TITLE
Add unused import remover to cleanup use imports after changes

### PR DESCRIPTION
**Pull request by TomasVotruba**
## Todo

- [x] make optinal, similar to importNames()
- [x] enable in class rename by default in tests

### CODE REVIEW

File: rules-tests/Renaming/Rector/Name/RenameClassRector/Fixture/class_annotations_target_entity.php.inc [hunk 1]
```diff
@@ -18,8 +18,6 @@ final class ClassAnnotationsTargetEntity
 
 namespace Rector\Tests\Renaming\Rector\Name\RenameClassRector\Fixture;
 
-use Doctrine\ORM\Mapping as ORM;
```
Comment (samsonasik) [inline rules-tests/Renaming/Rector/Name/RenameClassRector/Fixture/class_annotations_target_entity.php.inc]: The alias is used in annotation.

File: rules-tests/Renaming/Rector/Name/RenameClassRector/Fixture/rename_class.php.inc [hunk 2]
```diff
@@ -22,7 +22,7 @@ namespace MyNewNamespace;
 class MyNewClass
 {
     /**
-     * @return \MyNewNamespace\MyNewClass
+     * @return MyClass
```
Comment (samsonasik) [inline rules-tests/Renaming/Rector/Name/RenameClassRector/Fixture/rename_class.php.inc]: Just renamed class should be kept

File: rules-tests/Renaming/Rector/Name/RenameClassRector/Fixture/class_annotations_target_entity.php.inc (hunk 1 above)
Comment (TomasVotruba) [reply rules-tests/Renaming/Rector/Name/RenameClassRector/Fixture/class_annotations_target_entity.php.inc]: This is WIP, I'm working on those now

File: rules-tests/Renaming/Rector/Name/RenameClassRector/FixtureAutoImportNames/do_not_rename_docblock_inner_namespace.php.inc [hunk 3]
```diff
@@ -32,7 +32,6 @@ class DoNotRenameDocblockInnerNamespace
 namespace Rector\Tests\Renaming\Rector\Name\RenameClassRector\FixtureAutoImportNames;
 
 use Log;
-use CompanyB\Storage;
```
Comment (samsonasik) [inline rules-tests/Renaming/Rector/Name/RenameClassRector/FixtureAutoImportNames/do_not_rename_docblock_inner_namespace.php.inc]: Storage is used

File: rules-tests/Renaming/Rector/Name/RenameClassRector/Fixture/class_annotations_target_entity.php.inc (hunk 1 above)
Comment (samsonasik) [reply rules-tests/Renaming/Rector/Name/RenameClassRector/Fixture/class_annotations_target_entity.php.inc]: Ok, I think the PostRector should be enabled only when defined via `$rectorConfig`, eg:

```php
$rectorConfig->removeUnusedUses();
```

**Pull request merged**